### PR TITLE
Added placeholder image and fixed css issues

### PR DESCRIPTION
**Pull request by bradbieselin**

### CODE REVIEW

File: components/experience/index.tsx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+        <VerticalTimelineElement
+          className="vertical-timeline-element--education"
+          date="2021-Present"
+          iconStyle={{ background: "rgb(0, 0, 0)", color: "#fff" }}
+        >
+          <h3 className="vertical-timeline-element-title">
+            July 2021-Present: Witbe Customer Success Manager
+          </h3>
+          <h4 className="vertical-timeline-element-subtitle">New York, NY</h4>
+          <p></p>
```
Comment (Melincon) [inline components/experience/index.tsx]: Lets add content or remove this.

File: components/landing/index.tsx [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+  return (
+    <div className="Landing">
+      <div className="Landing__ImageContainer">
+        <img src="/Headshot.jpg" className="Landing__Image" />
+      </div>
+      <div className="Landing__TextContainer">
+        <div className="Landing__Text__Name">Brad Bieselin</div>
+        <div className="Landing__Text__Freelance">-Freelance Web Developer</div>
+        <div className="Landing__Text__Company">-Company: Witbe Inc</div>
+        <div className="Landing__Text__CompanyRole">
```
Comment (Melincon) [inline components/landing/index.tsx]: Stick to one naming convention here.

File: components/landing/index.tsx [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+          -Position: Project Manager
+        </div>
+        <div className="Landing__Text__School">
+          -2019 B.S. Computer Science Graduate
+        </div>
+        <div className="Landing__Text__Long">
```
Comment (Melincon) [inline components/landing/index.tsx]: Also these need to change.

File: components/landing/index.tsx [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+            font-size: 25px;
+          }
+          .Landing__Text__CompanyRole {
+            flex-wrap: wrap;
+            text-align: left;
+            color: white;
+            padding-bottom: 1rem;
+            font-size: 25px;
+          }
+          .Landing__Text__School {
```
Comment (Melincon) [inline components/landing/index.tsx]: Consistency with naming convention here as well

**Review by Melincon — CHANGES_REQUESTED** [review]

File: components/experience/index.tsx (hunk 1 above)
Comment (bradbieselin) [reply components/experience/index.tsx]: Added content to the p tag.

File: components/landing/index.tsx (hunk 2 above)
Comment (bradbieselin) [reply components/landing/index.tsx]: Changed naming convention for consistency

File: components/landing/index.tsx (hunk 3 above)
Comment (bradbieselin) [reply components/landing/index.tsx]: Changed naming convention for consistency

File: components/landing/index.tsx (hunk 4 above)
Comment (bradbieselin) [reply components/landing/index.tsx]: Changed naming convention for consistency

**Review by Melincon — APPROVED** [review]

**Pull request merged**